docs/04_contributing.md

```diff
@@ -59,7 +59,7 @@ $ <Iterate your development>
 $ git push fork <your_PR_branch>
 ```
 
-1. If your change affects users (adds,removes or changes a feature) add the item to the [CHANGELOG](CHANGELOG.md).
+1. If your change affects users (adds,removes or changes a feature) add the item to the [CHANGELOG](08_CHANGELOG.md).
```

Comment:
Hm, I think we should keep the CHANGELOG filename unchanged. Ideally we want it last in the docs list, but it is ok even if shows at the begining.